### PR TITLE
Add basic fort search logic

### PR DESCRIPTION
**Pull request by CapCap**
#95

### CODE REVIEW

**Comment by CapCap** [comment]
Eventually should refactor globally useful things like geo dist calc to it's own utility class, etc

File: cell_workers/seen_fort_worker.py [hunk 1]
```diff
@@ -90,9 +90,10 @@ def work(self):
                 print('may search too often, lets have a rest')
                 return 11
         time.sleep(8)
-        return 0
+        return
```
Comment (PokemonGoF) [inline cell_workers/seen_fort_worker.py]: Is this a typo?

File: bot.py [hunk 2]
```diff
@@ -37,13 +37,17 @@ def work_on_cell(self, cell, position):
                 worker.work()
         if self.config.spinstop:
             if 'forts' in cell:
+                # Only include those with a lat/long
+                forts = [fort for fort in cell['forts'] if 'latitude' in fort]
+                # Sort all by distance from current pos- eventually this should build graph & A* it
+                forts.sort(key=lambda x: SeenFortWorker.geocalc(self.position[0], self.position[1], fort['latitude'], fort['longitude'])) 
+                print forts
                 for fort in cell['forts']:
-                    if 'type' in fort:
```
Comment (PokemonGoF) [inline bot.py]: you need the 'type', or it will be a gym.

**Comment by vincentbeutin** [comment]
@CapCap this should be done also here ? : 

bot.py l.60
`for cell in map_cells:
     self.bot.work_on_cell(cell,position)`

For my initial example i think the nearest pokestop is not in the first cell.

What do you think ?

**Comment by CapCap** [comment]
@vincentbeutin I think you're absolutely right

**Comment by CapCap** [comment]
I also think we shouldn't split everything up like this in terms of distance calculation- it should go to the nearest pokemon/fort, not nearest pokemon then nearest forts

**Pull request merged**